# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xfsprogs/hotfix-non-glibc.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- xfsprogs-6.15.0/io/stat.c.vanilla	2025-07-24 16:48:32.447166221 +0200
+++ xfsprogs-6.15.0/io/stat.c	2025-07-24 16:49:49.089835234 +0200
@@ -21,6 +21,10 @@
 
 #include <fcntl.h>
 
+#ifndef STATX__RESERVED
+#define STATX__RESERVED            0x80000000U
+#endif
+
 static cmdinfo_t stat_cmd;
 static cmdinfo_t statfs_cmd;
 static cmdinfo_t statx_cmd;
